app/src/main/java/com/gathering/android/auth/signin/ForgetPasswordViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.gathering.android.auth.signin
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.gathering.android.auth.AuthRepository
+import com.gathering.android.auth.model.ResponseState
+import javax.inject.Inject
+
+class ForgetPasswordViewModel @Inject constructor(
+    private val repository: AuthRepository
+) : ViewModel() {
+
+    private val _viewState = MutableLiveData<ForgetPasswordViewState>()
+    val viewState: LiveData<ForgetPasswordViewState> by ::_viewState
+
+    fun onSendLinkBtnClicked(email: String) {
+        repository.resetPassword(email) { state ->
```

Comment:
are we verifying if the email is valid anywhere ?

Reply:
no. i will add a function called isEmailValid.